packages/aws-cdk-lib/aws-ec2/lib/vpc.ts

```diff
@@ -858,12 +879,22 @@ const NAME_TAG: string = 'Name';
  * Configuration for Vpc
  */
 export interface VpcProps {
+  /**
+   * The protocol of the vpc.
+   *
+   * Options are IPv4 only or dual stack.
+   *
+   * @default VpcProtocol.IPV4_ONLY
+   */
+  readonly vpcProtocol?: VpcProtocol;
```

Comment:
ipProtocol? 😜

Reply:
It is not really a protocol, as their is V4, V6, and dual stack. I guess you could have flags, but "none" is not really valid. 

There are also different configurations for subnets : right now a VPC can only be IPV4_ONLY or DUAL_STACK, whereas a subnet can be IPV4_ONLY, IPV6_ONLY, or DUAL_STACK.   You could still combine into a single enum, and maybe "ipProtocol" is okay (another suggestion was VpcMode) ... but then only 2 of the values would be valid for a VPC.